packages/inter-protocol/test/test-voPool.js

```diff
@@ -84,14 +85,24 @@ const voPoolTest = async (t, mutation, postTest) => {
     const storageNode = makeFakeStorage('voPoolTest');
     const marshaller = makeFakeMarshaller();
 
-    return definePoolKind(
+    /** @type {import('../src/vpool-xyk-amm/multipoolMarketMaker.js').AmmState} */
+    const ammState = {
       zcf,
+      // @ts-expect-error test unused
+      secondaryBrandToPool: undefined,
+      // @ts-expect-error test unused
+      secondaryBrandToLiquidityMint: undefined,
       centralBrand,
       timer,
       quoteIssuerKit,
-      // @ts-expect-error loose
-      paramAccessor,
+      // @ts-expect-error TS is confused
```

Comment:
this is because `makeAmmParamManager` above has the wrong return type. but it's not worth fixing.